deploy/deploy.gradle

```diff
@@ -88,7 +88,10 @@ afterEvaluate { project ->
                         }
 
                         final depGroup = dep.group
-                        final depName = dep.name
+                        // If it's a project dependency, use the artifactId, otherwise use its name.
```

Comment:
without this change, the dep.name will return the last portion of the module name path -   'googleplay'/'default'/'cardscan' instead of 'ml-core-googleplay'/'ml-core-default'/'ml-core-cardscan', resulting in pom resolution error